mod/beacon/blockchain/payload.go

```diff
@@ -137,10 +128,10 @@ func (s *Service[
 		// and possibly should be made more explicit later on.
 		lph.GetParentHash(),
 	); err != nil {
-		s.metrics.markRebuildPayloadForRejectedBlockFailure(slot, err)
+		s.metrics.markRebuildPayloadForRejectedBlockFailure(stateSlot, err)
```

Comment:
slot is instantiated and never written. I believe we want to log the slot we are trying to build a block for.